Sim/SimG4Components/src/SimG4Svc.cpp

```diff
@@ -64,13 +64,14 @@ StatusCode SimG4Svc::initialize() {
   }
 
   m_runManager.Initialize();
+
+  std::unique_ptr<G4VisManager> visManager(new G4VisExecutive);
```

Comment:
if the G4VisManager has to survive until the end of the program it cannot be a unique_ptr in the local scope.

Reply:
For the case that I am using (running DAWN), the pointer survives until the end of the program and there is no segmentation fault. Should I then follow the example of the RunManager?

Reply:
Exactly, if the lifetime for the Visualisation Manager is the same as for the SimG4Svc, then it should move out of the local scope and be a member of SimG4Svc, so it has ownership.